tsconfig-build.json

```diff
@@ -35,4 +36,4 @@
         "flatModuleOutFile": "angular-library-starter.js",
         "flatModuleId": "angular-library-starter"
     }
-}
\ No newline at end of file
+}
```

Comment:
The extension of the tsconfig file is usually done to avoid rewriting shared properties. In this case the files are completely different (the basic tsconfig is just for dev & testing) and I see no benefits in adding the extension.